### PR TITLE
Week 2: servlet messages

### PR DESCRIPTION
**Pull request by kevinkong03**

### CODE REVIEW

**Review by rjkroege — COMMENTED** [review]
Seems fine.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 1]
```diff
@@ -1,18 +1,27 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```
Comment (rjkroege) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java]: Imports should be in some well defined order. Perhaps alphabetical.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -29,7 +29,14 @@ function addMusic() {
 
 async function showMessage() {
 const responseFromServer = await fetch('/hello');
+const textFromResponse = await responseFromServer.json;
+let pick = Math.floor(Math.random() * textFromResponse.length);
+const helloContainer = document.getElementById("hello-container");
+helloContainer.innerText = textFromResponse[pick];
+
+/*
```
Comment (rjkroege) [inline portfolio/src/main/webapp/script.js]: Better to remove commented out code. The benefit of keeping the world in `git` is that you can always inspect the old version of the file.

**Review by rjkroege — COMMENTED** [review]
PR seems fine but per other blither about structuring the PRs.

**Comment by rjkroege** [comment]
What order am I suppose to review in?

**Comment by kevinkong03** [comment]
Not sure why force pushing added five commits instead of just one, but I think it's okay...

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,12 @@ function addMusic() {
   const musicContainer = document.getElementById('music-container');
   musicContainer.innerText = musician;
 }
+
+async function showMessage() {
+const responseFromServer = await fetch('/hello');
```
Comment (rjkroege) [inline portfolio/src/main/webapp/script.js]: You should indent this? This code is inside the { yes?

**Review by rjkroege — CHANGES_REQUESTED** [review]

**Review by rjkroege — APPROVED** [review]

**Pull request merged**